spisea/ifmr.py

```diff
@@ -1,27 +1,677 @@
 ##########################################################
 #
 #
-#This IFMR comes from Raithel et al. 2017
```

Comment:
For all these, include the ads link to the published version
I think Kalirai is 2008